tariff/awattar.go

```diff
@@ -62,8 +62,14 @@ func (t *Awattar) run(done chan error) {
 	for ; true; <-tick.C {
 		var res awattar.Prices
 
+		// Awattar publishes prices for next day around 13:00 CET/CEST, so up to 35h of price data are available
+		// To be on the safe side request a window of -2h and +48h, the API doesn't mind requesting more than available
```

Comment:
Should we round to hours and do we really need two hours back instead of start of current hour?

Reply:
Not sure if we need 2 hours, but it could mitigate potential edge cases, like some potential errors in the awattar api regarding daylight savings time. Requesting one entry more doesn't seem worth investigating potential edge cases.

Start time in the Awattar api is exclusive, requesting start=12:15 for example returns 13:00-14:00 as the first entry in the response.

Rounding would be possible, but I don't see the benefit, except of adding complexity.

The PR is open for your changes.